doc/topics/development/labels.rst

```diff
@@ -0,0 +1,287 @@
+    in Salt for tracking purposes. (This can be a stand-alone label.)
+
+Won't Fix for Now
+    The SaltStack team has acknowledged the issue at hand is legitimate, but made the call that it’s not something
+    they’re able or willing to fix at this time. These issues may be revisited in the future.
+
+Other
+~~~~~
+
+There are a couple of other labels that are helpful in categorizing bugs that are not included in the categories above.
+These labels can either stand on their own such as ``Question`` or can be applied to bugs or feature requests as
+applicable.
+
+Low Hanging Fruit
+    Applied to bugs that should be easy to fix. This is useful for new contributors to know where some simple things
+    are to get involved in contributing to salt.
+
+Question
+    Used when the issue isn’t a bug nor a feature, but the user has a question about expected behavior, how something
+    works, is misunderstanding a concept, etc. This label is typically applied on its own with ``Blocked`` milestone.
+
+Regression
+    Helps with additional filtering for bug fixing. If something previously worked and now does not work, as opposed to
+    something that never worked in the first place, the issue should be treated with greater urgency.
+
+ZD
+    Stands for “Zendesk” and is used to help track bugs that customers are seeing as well as community members. Bugs
+    with this label should be treated with greater urgency.
+
+Pull Request Labels
+-------------------
+
+SaltStack also applies various labels to incoming pull requests. These are mainly used to help SaltStack engineers
+easily identify the nature the changes presented in a pull request and whether or not that pull request is ready to be
+reviewed and merged into the Salt codebase.
+
+Type of Change
+~~~~~~~~~~~~~~
+
+A "* Change" label is applied to each incoming pull request. The type of change label that is applied to a pull request
+is based on a scale that encompasses the number of lines affected by the change in conjunction with the area of code
+the change touches (i.e. core code areas vs. execution or state modules).
+
+The conditions given for these labels are recommendations, as the pull request reviewer will also consult their
+intuition and experience regarding the magnitude of the impact of the proposed changes in the pull request.
+
+Core code areas include: state compiler, crypto engine, master and minion, transport, pillar rendering, loader,
+transport layer, event system, salt.utils, client, cli, logging, netapi, runner engine, templating engine, top file
+compilation, file client, file server, mine, salt-ssh, test runner, etc.
+
+* Minor Change
+    * Less than 64 lines changed, or
+    * Less than 8 core lines changed
+* Medium Change
+    * Less than 256 lines changed, or
+    * Less than 64 core lines changed
+* Master Change
+    * More than 64 lines changed, or
+    * More than 256 core lines changed
```

Comment:
Sorry to be picky but these should be reversed
    * More than 256 lines changed, or
    * More than 64 core lines changed